### ISSUE TITLE
--auto-vertical-output support?

### ISSUE THREAD

**Issue by michaelstack** [dbcli/mycli#195]
Does mycli have any way to get behaviour similar to mysql's --auto-vertical-output option?

I tried adding it when I start mycli, but got an error:

```
Error: no such option: --auto-vertical-output
```

**Comment by amjith** [comment]
We currently don't have this feature available. But it is doable. I'll let you know when the feature is implemented.

**Comment by michaelstack** [comment]
Thanks.

Looking forward to it!

**Comment by amjith** [comment]
Released as part of 1.6.0. Please upgrade: 

```
pip install -U mycli 
```
